BugSplatDotNetStandard/BugSplat.cs

```diff
@@ -2,8 +2,10 @@
 using System.Collections.Generic;
 using System.IO;
 using System.Net.Http;
+using System.Net.Http.Headers;
 using System.Threading.Tasks;
 using BugSplatDotNetStandard.Utils;
+using Newtonsoft.Json;
```

Comment:
This does add a dependency for de-serialization of the AWS URL object. Newtonsoft is recommended by .Net. That said, if we still want to avoid this it's possible for me to add custom de-serialization.

Reply:
Newtonsoft is a well worn path, this is the correct approach 🙂